### PR TITLE
Update README with an example that works

### PR DESCRIPTION
**Pull request by steiza**
#### Summary

I was doing some testing with sigstore-go, and I noticed the README examples were out of date, so I think we should update them.

#### Release Note

NONE

#### Documentation

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,26 +27,37 @@ Sigstore already has a canonical Go client implementation, [cosign](https://gith
...
+}
 ```
 
-```bash
-$ go run cmd/sigstore-go/main.go -tufRootURL tuf-repo-cdn.sigstore.dev examples/bundle-provenance.json
-Verification successful!
-```
+You can also specify a TUF root with something like `-tufRootURL tuf-repo-cdn.sigstore.dev`.
 
 Alternatively, you can install a binary of the CLI like so:
```
Comment (Hayden-IO) [inline README.md:56]: Is this binary meant for typical users or just for testing?

Comment (steiza) [reply README.md:56]: That's correct - the binary is to illustrate how to use the library, and is not intended to be fully functional like other Sigstore CLI tools (e.g. `cosign`).

Comment (Hayden-IO) [reply README.md:56]: Do we want to make a note of that in the README?

Comment (steiza) [reply README.md:56]: ... of course, after I push additional language, I notice right below it we also have:

> Sigstore already has a canonical Go client implementation, [cosign](https://github.com/sigstore/cosign), which was developed with a focus on container image signing/verification. It has a rich CLI and a long legacy of features and development. `sigstore-go` is a more minimal and friendly API for integrating Go code with Sigstore, with a focus on the newly specified data structures in [sigstore/protobuf-specs](https://github.com/sigstore/protobuf-specs).

Comment (Hayden-IO) [reply README.md:56]: I think your clarification helps - The quote does not explicitly say that sigstore-go as a CLI is discouraged.

**Review by Hayden-IO — APPROVED** [review]

**Pull request merged**